### DISCUSSION TITLE
Apicurio Registry authentication using PING Identity and OpenID Connect

### DISCUSSION THREAD

**Discussion by arthurguau** [Apicurio/apicurio-registry#2249] [Q&A] [answered]
If I am not wrong, KeyCloak is the officially supported IAM server, as some KeyCload related parameters are presented in source code.  But not all the organisations use KeyCloak as security platform.  Is it possible to use something else other than KeyCloak, like PING Identity?

**Comment by carlesarnal** [comment, marked as answer]
Keycloak is the only IAM supported server for now. It's in our backlog to support other OIDC providers but, for now, the application only works with Keycloak.

**Reply by Gourav2912** [reply]
@carlesarnal @EricWittmann Is this valid and only OIDC support is with KeyCloak?

**Reply by carlesarnal** [reply]
Yes, this is still true, but it depends on your needs. The existing problem is with the UI, the backend can work just fine with any oidc provider.

**Reply by alexisph** [reply]
Hi @carlesarnal . Can you please explain how it can work with any OIDC provider?

**Reply by carlesarnal** [reply]
Hi alexisph, please, if you have questions regarding the configuration, please, open a new question/discussion/issue. Thanks!
